pkg/jobrunaggregator/jobrunaggregatorapi/types_row_alert.go

```diff
@@ -64,9 +66,14 @@ INNER JOIN openshift-ci-data-analysis.ci_data.Jobs on JobRuns.JobName = Jobs.Job
 )
 
 type AlertRow struct {
-	JobRunName   string
-	Name         string
-	Namespace    string
-	Level        string
-	AlertSeconds int
+	Name            string
+	Namespace       string
+	Level           string
+	AlertSeconds    int
+	JobName         string
+	JobRunName      string
+	JobRunStartTime time.Time
+	JobRunEndTime   time.Time
+	Cluster         string
+	ReleaseTag      string
 }
```

Comment:
One thing that I have found when adding columns to existing tables is that they have to be Nulable if we are going to read existing values that don't have the new data.

https://pkg.go.dev/cloud.google.com/go/bigquery#InferSchema
https://stackoverflow.com/questions/51040470/exception-after-mapping-bigquerys-rows-with-null-to-struct

Reply:
Thanks, working on it.

Reply:
These schemas should all be deleted IMO, they're virtually all out of date and we can't kid ourselves into thinking we'll ever keep them up to date. They're unused, eternally out of date, buying us nothing but more work and confusion. If we ever need programatic schema management, we should do it properly, but today I think these should just be deleted in another PR personally.

Reply:
Looking at those, I have to agree; they are not used and if you know where the tables live in big query, that doc is kind of superfluous because it's that same info is right there in the schema section.  I also see we add some code just to make the linter happy.

Reply:
Ok the PR is updated with bigquery null types, tests are passing, and I've run an alert upload locally, you can now see values where (for example) cluster is populated in the db tables.

Reply:
hack/run-job-aggregator-analyzer.sh seems to run ok, that should be a good indicator of reading. A lot of those might use views though so it's hard to say how we'd expose this. 

Once we do the table swap there shouldn't be any more nulls, unless perhaps the source job run had nulls. 

I think we're ok to go.